### PR TITLE
94 status in edit

### PR DESCRIPTION
**Pull request by cstanger**

### CODE REVIEW

**Comment by cstanger** [comment]
add status dropdown to edit field 
and change delete function to change service status

**Comment by cstanger** [comment]
see https://github.com/MaibornWolff/cloud-landscape/pull/95

**Comment by cstanger** [comment]
it is merged to https://github.com/MaibornWolff/cloud-landscape/pull/95

File: src/map/components/detailModal/deleteDialog.component.tsx [hunk 1]
```diff
@@ -64,8 +64,8 @@ export default function DeleteDialog(props: Props) {
           </DialogTitle>
           <DialogContent>
             <DialogContentText>
-              The service will be deleted from the database and not be available
-              to the visitors of the site any more. This can not be undone. Are
+              The service is NOT deleted from the database. Only the status is
+              set to deleted and the service is no longer displayed publicly.
               you sure you want to proceed?
```
Comment (elKei24) [inline src/map/components/detailModal/deleteDialog.component.tsx:69]:
```suggestion
              Are you sure you want to proceed?
```

**Pull request merged**